_layouts/default.html

```diff
@@ -51,12 +53,20 @@
 
 
     {% if page.url != "/" %}
-    <div class="name">
+    <!-- <div class="name">
         <div class="container">
             <a href="/">
                 <h1>Frost Music &amp; Arts Festival</h1>
             </a>
         </div>
+    </div> -->
+    <div class="banner-container">
+        <!-- <img class="banner" src="/assets/img/21/blank-banner.jpeg" />  -->
+        <div class="banner-frost-name-container">
```

Comment:
What is this blank banner? Just want to check that it's approved by the artists.